README.md

```diff
@@ -116,6 +117,31 @@ bom: manufacture/advanced-example-BOM.tsv
 gerbers: manufacture/gerbers-and-drills
 ```
 
+#### The multi field
+Kitspace supports multiple projects in one repository with the `multi` field. When multiple projects exist, `multi` will always be the first field in the `kitspace.yaml`, with the paths to your projects folder nested underneath.
+
+```
+├── kitspace.yaml
+├── project_one
+│    ├── 1-click-bom.tsv
+│    └── gerbers
+├── project_two
+     ├── 1-click-bom.tsv
+     └── gerbers
+
+```
+with `kitspace.yaml` containing:
+```yaml
+multi:
+    project_one:
+        summary: First project in a repository.
+        color: blue
+        site: https://example-one.com
+    Project_two:
```

Comment:
`s/Project/project/`

Reply:
Can you expand on that? 

I see I capitalized `project_two` on accident. Add one more folder layer?

Reply:
no no, just that capitalization.

Reply:
Cool, got it fixed :+1: